### PR TITLE
Fix duplicate operationIDs in Galaxy API

### PR DESCRIPTION
**Pull request by gerrod3**
fixes: #1356

### CODE REVIEW

File: pulp_ansible/app/galaxy/views.py [hunk 1]
```diff
@@ -189,6 +190,7 @@ def post(self, request, path):
         return OperationPostponedResponse(async_result, request)
 
 
+@extend_schema_view(get=extend_schema(operation_id="api_v2_collection_versions_list"))
```
Comment (mdellweg) [inline pulp_ansible/app/galaxy/views.py:193]: I'm not too good glad that the liveapi is part in our open API, but I guess it's too late anyway.

**Review by mdellweg — APPROVED** [review]
Approving here, but can we think about a way to prefix all APIs in the galaxy part?

**Comment by gerrod3** [comment]
> Approving here, but can we think about a way to prefix all APIs in the galaxy part?

What type of prefix would you want?

**Comment by gerrod3** [comment]
@jlsherrill Do you need these backported to 0.16?

**Comment by jlsherrill** [comment]
i don't really need a backport, but being included in a release sooner rather than later would be helpful.  How long till 0.17?

**Comment by gerrod3** [comment]
> i don't really need a backport, but being included in a release sooner rather than later would be helpful. How long till 0.17?

It will be release when Repo cross search and rbac are merged. It is also requires pulpcore 3.23, so sometime in March.

**Comment by mdellweg** [comment]
`galaxy_api_v1/2` would be compelling, no?
or just `galaxy_v1/2`. We know we are in an api definition at that point.

**Comment by jlsherrill** [comment]
that time frame should be okay, no backport needed.

**Pull request merged**